### ISSUE TITLE
cannot inherit theme

### ISSUE THREAD

**Issue by gllo** [tranduyhung/grav-theme-clean-blog#12]
I want to make modifications to clean-blog and followed grav tutorial how to make child theme.

crystal.yaml:
`streams:  
   schemes:
   theme:
     type: ReadOnlyStream
       prefixes:
         '':
           - user/themes/crystal
           - user/themes/clean-blog`

blueprints.yaml:
`name: Crystal
version: 1.0.0
description: "Extending clean-blog"`

crystal.php:
`<?php
namespace Grav\Theme;

class Crystal extends Cleanblog
{
   // Some new methods, properties etc.
}
?>`

When I try this I get the error that 
`Class 'Grav\Theme\Cleanblog' not found`

I then changed to "clean-blog" the name of the class (to match folder name) but the it cannot tolerate hyphen then.

How can I extend clean-blog theme?

**Comment by tranduyhung** [comment]
I haven't tried by myself but you extend from Cleanblog class but you don't include it anywhere in your PHP file, you may want to try:

````
<?php
namespace Grav\Theme;

use Grav\Theme\Cleanblog;

class Crystal extends Cleanblog
{
// Some new methods, properties etc.
}
?>

**Comment by gllo** [comment]
Thanks, but it is not working I always get "Class 'Grav\Theme\Cleanblog' not found"
I already tried including in the child theme crystal.php:
`use Grav\Common\Theme\Cleanblog;`
and also
`use Grav\Theme\Cleanblog;`
with no success.
Also cleared full grav and browser cache, just to be sure.

Earlier I inherited another theme without including the direct parent in the child php file.

I am full of problems, the other one is, that by default your original clean-blog is not showing the default header image in the header (img/head-bg.jpg). I read in other guys question here, that I should use header_image variable. Can I put it in clean-blog.yaml? How can I define the proper URL to an image? What is the root folder that should be taken into account?

**Comment by gllo** [comment]
Never mind my second problem. After  reading some code I've figured it out. I have the possibility to edit stuff defined in blueprints.yaml (e.g. copyright, default_header_image etc.). I have put `default_image_header` into my `/user/config/clean-blog.yaml`. I have assigned the name of the background image to it, simply just the plain name of the file without path. I don't need path since the image name is automagically searched for in the `theme:\\img` folder. So it is expected that the background image be put into th `img/` folder if I am correct (or have to also edit the twig).
